content/introduction/use-cases/offline-first.md

```diff
@@ -20,6 +20,12 @@ When it is safe to connect to the internet, Dolt computes the difference between
 
 Conflicting writes are surfaced quickly and an operator or software can take additional action to resolve.
 
+# Dolt replaces
+
+## Custom syncing processes
+
+Dolt replaces custom code to synchronize your client and server. This code is complicated and hard to get right. The Git [remote](../../concepts/dolt/git/remotes.md) model of clone, fetch, push, and pull is a proven synchronization model. Dolt brings this model to the database allowing you to remove most of your synchronization code. 
```

Comment:
CRDT is a popular engineering buzzword/solution in this space, but CRDTs still resolve conflicts in ways that stomp conflicts. Sometimes that's a good thing, sometimes you don't want to stomp conflicts.

Reply:
This is a good one. I'll have to think about it though.